frontend/test/__runner__/backend.js

```diff
@@ -22,10 +24,10 @@ export const BackendResource = createSharedResource("BackendResource", {
   create({ dbKey = DEFAULT_DB_KEY }) {
     const dbFile = getDbFile();
     const absoluteDbKey = dbKey ? __dirname + dbKey : dbFile;
-    if (process.env["E2E_HOST"] && dbKey === DEFAULT_DB_KEY) {
+    if (e2eHost) {
```

Comment:
The `dbKey` that gets passed in when running normally is `''` instead of being `DEFAULT_DB_KEY`